pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStreamingDispatcherMultipleConsumers.java

```diff
@@ -155,6 +155,11 @@ protected void cancelPendingRead() {
 
     @Override
     public synchronized void readMoreEntries() {
+        if (sendInProgress) {
```

Comment:
This will have a big performance impact.
Also the subscription may become stuck.
If you skip this round you have to rescheule the execution

I can't double check it now, but I am not sure that this is the right fix

Reply:
Hi @eolivelli 
Can you clarify this problem?
It looks like this logic already exists for `PersistentDispatcherMultipleConsumers`.